src/mentors.json

```diff
@@ -1208,6 +1208,35 @@
       }
     ]
   },
+  {
+    "id": "kenichi@shibata.co.uk",
+    "name": "Kenichi Shibata",
+    "avatar": "https://avatars1.githubusercontent.com/u/10043789?s=460&v=4",
+    "title": "Platform (Application) Architect @Comparethemarket.com",
```

Comment:
Now the "title" and "description" is too long :(
We have no enough space for longer fields. Sorry..
https://travis-ci.com/Coding-Coach/find-a-mentor/builds/104780568#L475
If you do this locally, just run `yarn test` and make sure it pass.